collector.proto

```diff
@@ -40,15 +40,15 @@ message Span {
     string operation_name = 2;
     repeated Reference references = 3;
     google.protobuf.Timestamp start_timestamp = 4;
-    uint32 duration_micros = 5;
+    uint64 duration_micros = 5;
```

Comment:
I'll live with it :)

I *am* against 96-bit durations, though (i.e., `google.protobuf.Duration`).